src/middlewares/validate.js

```
@@ -1,20 +1,79 @@
+    try {
+      let data;
+
+      // Get data based on the source
+      switch (source) {
+        case 'params':
+          data = c.req.param();
+          break;
+        case 'query':
+          data = c.req.query();
+          break;
+        case 'json':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)